sympy/utilities/lambdify.py

```diff
@@ -102,10 +102,10 @@
 NUMEXPR_TRANSLATIONS = {}
 
 # Available modules:
-MODULES = {
+_MODULES = {
     "math": (MATH, MATH_DEFAULT, MATH_TRANSLATIONS, ("from math import *",)),
     "mpmath": (MPMATH, MPMATH_DEFAULT, MPMATH_TRANSLATIONS, ("from mpmath import *",)),
-    "numpy": (NUMPY, NUMPY_DEFAULT, NUMPY_TRANSLATIONS, ("import_module('numpy')",)),
+    "numpy": (NUMPY, {}, {}, ("import numpy",)),
```

Comment:
Should we make the translation dictionaries empty by default now, since the actual default translations are done in the printers now?